src/SWP/Bundle/ContentBundle/Controller/RouteToArticleController.php

```diff
@@ -0,0 +1,166 @@
+ * @license http://www.superdesk.org/license
+ */
+namespace SWP\Bundle\ContentBundle\Controller;
+
+use FOS\RestBundle\Controller\FOSRestController;
+use FOS\RestBundle\View\View;
+use SWP\Bundle\ContentBundle\Form\Type\RouteToArticleType;
+use SWP\Bundle\ContentBundle\Model\RouteToArticle;
+use Nelmio\ApiDocBundle\Annotation\ApiDoc;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Method;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\ParamConverter;
+use Symfony\Component\HttpFoundation\Request;
+
+class RouteToArticleController extends FOSRestController
+{
+    /**
+     * List all RouteToArticle entities for current tenant.
+     *
+     * @ApiDoc(
+     *     resource=true,
+     *     description="List all RouteToArticleEntities for current tenant",
+     *     statusCodes={
+     *         200="Returned on success.",
+     *     }
+     * )
+     * @Route("/api/{version}/content/routetoarticle/", options={"expose"=true}, defaults={"version"="v1"}, name="swp_api_content_list_routetoarticle")
+     * @Method("GET")
+     */
+    public function listAction(Request $request)
+    {
+        $entityManager = $this->get('doctrine')->getManager();
+        $paginator = $this->get('knp_paginator');
+        $routeToArticles = $paginator->paginate($entityManager->getRepository('SWPContentBundle:RouteToArticle')->findAll());
+
+        if (count($routeToArticles) == 0) {
+            throw new NotFoundHttpException('RouteToArticles were not found.');
+        }
+
+        return $this->handleView(View::create($this->container->get('swp_pagination_rep')->createRepresentation($routeToArticles, $request), 200));
+    }
+
+    /**
+     * Get single RouteToArticle Entity.
+     *
+     * @ApiDoc(
+     *     resource=true,
+     *     description="Get single RouteToArticle Entity",
+     *     statusCodes={
+     *         200="Returned on success.",
+     *         404="Entity not found",
+     *     }
+     * )
+     * @Route("/api/{version}/content/routetoarticle/{id}", requirements={"id"="\d+"}, options={"expose"=true}, defaults={"version"="v1"}, name="swp_api_content_get_routetoarticle")
+     * @Method("GET")
+     * @ParamConverter("routeToArticle", class="SWPContentBundle:RouteToArticle")
+     */
+    public function getAction(RouteToArticle $routeToArticle)
+    {
+        return $this->handleView(View::create($routeToArticle, 200));
+    }
+
+    /**
+     * Create new RouteToArticle Entity.
+     *
+     * @ApiDoc(
+     *     resource=true,
+     *     description="Create new RouteToArticle Entity",
+     *     statusCodes={
+     *         201="Returned on success.",
+     *         400="Returned when form has errors"
+     *     },
+     *     input="SWP\Bundle\ContentBundle\Form\RouteToArticleType"
+     * )
+     * @Route("/api/{version}/content/routetoarticle", options={"expose"=true}, defaults={"version"="v1"}, name="swp_api_content_create_routetoarticle")
```

Comment:
I would change url for all actions in controller to ```/api/{version}/mapping/routes``` Somthing like that - to don't mix with content. As this will be not so often used feature.